contracts/base/TokenCollateralLoans.sol

```diff
@@ -147,6 +147,24 @@ contract TokenCollateralLoans is LoansBase {
         collateralTokenTransferFrom(msg.sender, amount);
     }
 
+    function _requireExpectedBalance(uint256 initialBalance, uint256 expectedAmount)
```

Comment:
It seems like it overlaps with changes in #70 , let's make sure that we use the updated function here, after that PR is merged.